manim/mobject/types/vectorized_mobject.py

```diff
@@ -2116,8 +2127,60 @@ def construct(self):
                     self.play( # Animate group without component
                         (gr-circle_red).animate.shift(RIGHT)
                     )
+
+        A `VGroup` can be created using iterables as well. Keep in mind that all generated values from an
+        iterable must be an instance of `VMobject`. This is demonstrated below:
+
+        .. manim:: AddIterableToVGroupExample
+            :save_last_frame:
+
+            class AddIterableToVGroupExample(Scene):
+                def construct(self):
+                    v = VGroup(
+                        Square(),               # Singular VMobject instance
+                        [Circle(), Triangle()], # List of VMobject instances
+                        Dot(),
+                        (Dot() for _ in range(2)), # Iterable that generates VMobjects
+                    )
+                    v.arrange()
+                    self.add(v)
```

Comment:
Maybe it would be helpful to note after the example that it works by unpacking iterables, so indexing a `VGroup` won't give you an iterable.

Reply:
All done, but let me know if I could be more clearer with my explanation! Thanks so much for all your feedback :)